driver/main_args.ml

```diff
@@ -83,6 +83,12 @@ let mk_dllpath f =
   "<dir>  Add <dir> to the run-time search path for shared libraries"
 ;;
 
+let mk_stop_after f =
+  "-stop-after", Arg.String f,
```

Comment:
We could also use `Arg.Symbol (Compile_pass.names, f)`, this way we don't need to validate the input in `f`